docs/guides/Checkout/creating-a-regular-order-from-an-existing-cart.mdx

```diff
@@ -0,0 +1,684 @@
+---
+title: "Creating a regular order from an existing cart"
+slug: "creating-a-regular-order-from-an-existing-cart"
+hidden: false
+createdAt: "2023-07-18T17:08:52.219Z"
+updatedAt: "2023-07-18T17:08:52.219Z"
+excerpt: "Learn how to use VTEX APIs to place a regular order from an existing cart."
+---
+
+This step-by-step guide will walk you through the process of using the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api) to create an order from an existing shopping cart. This process is divided into the following steps:
+
+1. **Get cart information:** The process begins with retrieving the `orderFormId` of the current shopping cart via the [Get current or create a new cart](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pub/orderForm) endpoint.
+2. **Place the order:** The process continues by placing the order on the VTEX platform via the [Place order from an existing cart](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/transaction) endpoint.
+3. **Handling the order payment and processing:** The process ends by sending the necessary payment data to resolve the order payment and processing order details.
```

Comment:
Sugestão:
Para manter uma padronização de referenciar links (como foi feito nos dois primeiros passos), aqui no passo 3 seria interessante adicionar os links para os endpoints de pagamento e processamento da ordem.
Ex.  [sending the necessary payment data](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pub/transactions/-transactionId-/payments)  e  [processing order](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/gatewayCallback/-orderGroup-)